packages/one-app-bundler/webpack/loaders/validate-required-externals-loader.js

```diff
@@ -47,6 +47,12 @@ if (!global.BROWSER) {
 }
 `;
 
+    // NOTE: This is temporary. Since we only need 'requiredExternals' in module-config.json
+    //       we create, for now, the file right here with the data.
+    fs.writeFileSync(path.resolve(process.cwd(), 'build', packageJson.version, 'module-config.json'), JSON.stringify({
```

Comment:
I didn't want to until we actually work on the final implementation since it will change for sure, unless we believe that it will make a difference when we actually start working on it